migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration.go

```diff
@@ -31,20 +46,70 @@ var (
 				return errors.Wrap(err,
 					"moving simple_access_scopes from rocksdb to postgres")
 			}
+			legacyPermissionSetStore, err := legacypermissionsets.New(databases.PkgRocksDB)
+			if err != nil {
+				return err
+			}
+			if err := movePermissionSets(databases.GormDB, databases.PostgresDB, legacyPermissionSetStore); err != nil {
+				return errors.Wrap(err,
+					"moving permission_sets from rocksdb to postgres")
+			}
+			legacyRoleStore, err := legacyroles.New(databases.PkgRocksDB)
+			if err != nil {
+				return err
+			}
+			if err := moveRoles(databases.GormDB, databases.PostgresDB, legacyRoleStore); err != nil {
+				return errors.Wrap(err,
+					"moving roles from rocksdb to postgres")
+			}
 			return nil
 		},
 	}
 	batchSize = 10000
 	schema    = pkgSchema.SimpleAccessScopesSchema
 	log       = loghelper.LogWrapper{}
+
+	accessScopeIDMapping = map[string]string{
+		"denyall":      "ffffffff-ffff-fff4-f5ff-fffffffffffe",
+		"unrestricted": "ffffffff-ffff-fff4-f5ff-ffffffffffff",
+	}
+
+	permissionsetIDMapping = map[string]string{
+		"admin":                 "ffffffff-ffff-fff4-f5ff-ffffffffffff",
+		"analyst":               "ffffffff-ffff-fff4-f5ff-fffffffffffe",
+		"continuousintegration": "ffffffff-ffff-fff4-f5ff-fffffffffffd",
+		"none":                  "ffffffff-ffff-fff4-f5ff-fffffffffffc",
+		"scopemanager":          "ffffffff-ffff-fff4-f5ff-fffffffffffb",
+		"sensorcreator":         "ffffffff-ffff-fff4-f5ff-fffffffffffa",
+		"vulnmgmtapprover":      "ffffffff-ffff-fff4-f5ff-fffffffffff9",
+		"vulnmgmtrequester":     "ffffffff-ffff-fff4-f5ff-fffffffffff8",
+		"vulnreporter":          "ffffffff-ffff-fff4-f5ff-fffffffffff7",
+	}
 )
 
+func convertAccessScopeID(accessScopeID string) string {
+	identifierSuffix := strings.TrimPrefix(accessScopeID, accessScopeIDPrefix)
+	replacement, found := accessScopeIDMapping[identifierSuffix]
+	if found {
+		return replacement
+	}
```

Comment:
nitpick: let's use one-line if.
```suggestion
	if replacement, found := accessScopeIDMapping[identifierSuffix]; found {
		return replacement
	}
```

The same can be applied to `convertPermissionSetID`.